server/util/aria.test.js

```diff
@@ -79,3 +83,97 @@ describe('Verify test hashes are matching as expected', () => {
         expect(testsHashA).not.toEqual(testsHashB);
     });
 });
+
+describe('Verify test hashes are matching as expected - (v2 test format)', () => {
```

Comment:
Huge thanks for expanding these tests so much! Hashes are such delicate flowers that it really eases my mind a lot to know we have some extensive testing in place to make sure they keep working.